client-js/app/elements/labrad-grapher-dataset.html

```diff
@@ -18,15 +30,17 @@
   Polymer({
     is: 'labrad-grapher-dataset',
     properties: {
-      items: {
-        type: Array,
+      path: {
+        type: String,
         notify: true,
+      },
+      dataset: {
+        type: String,
+        notify: true
       }
     },
-    ready: function() {
-      this.items = [
-        'A single dataset'
-      ];
+    back: function(){
+      window.location.href='/grapher/'+this.path.toString().replace(/,/g,"/");
```

Comment:
I'm just going to put the paper button in```app-link```  anchor tags so it's exactly like all the other links. That way we can more easily play with the styles and appearance when we're polishing the site.